src/Kernel32/Kernel32.cs

```diff
@@ -141,7 +141,7 @@ public static partial class Kernel32
         /// If the function succeeds, the return value is the number of TCHARs stored in the output buffer, excluding the terminating null character.
         /// If the function fails, the return value is zero. To get extended error information, call <see cref="GetLastError"/>.
         /// </returns>
-        [DllImport(api_ms_win_core_localization_l1_2_0, CharSet = CharSet.Unicode, SetLastError = true)]
+        [DllImport(api_ms_win_core_localization_l1_2_0, CharSet = CharSet.Unicode, EntryPoint = "FormatMessageW", ExactSpelling = true, SetLastError = true)]
```

Comment:
You dont need ExactSpelling = true

Reply:
Thanks for reviewing. It doesn't seem to do any harm either, so I'm going to leave it as a way of expressing that I add the `W` suffix so the system doesn't need to try.
I hope this is just a temporary workaround anyway till the Windows SDK is fixed.

Reply:
Yes , we have a thread going with windows and there are more suffix free APIs that they added with no exports. Plan is to fix it by 4/6

Reply:
=> temporary workaround anyway till the Windows SDK is fixed.  
Forgot to mention that , using explicitly W or A is a good practice and i believe the  core recommendation is to use "W" always. So even after windows fix it , you should consider keeping this change.